ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -489,20 +465,6 @@ if (( regenConcatHP )); then
 				log_Msg "Run interpreted MATLAB/Octave (${interpreter[@]}) with code..."
 				log_Msg "${matlab_code}"
 
-				# Use bash redirection ("here-string") to pass multiple commands into matlab
-				# (Necessary to protect the semicolons that separate matlab commands, which would otherwise
-				# get interpreted as separating different bash shell commands)
-				# See note below about why we export FSL_FIX_WBC after sourcing FSL_FIXDIR/settings.sh
-				(
-					#fix's default settings.sh isn't safe to unbound variables or exit codes, so disable everything before sourcing
-					debug_disable_trap
-					set +u
-					source "${FSL_FIXDIR}/settings.sh"
-					set -u
-					debug_enable_trap
-					export FSL_FIX_WBC="${Caret7_Command}"
-					"${interpreter[@]}" <<<"${matlab_code}"
-				)
```

Comment:
The deleted code here isn't replaced with anything.  Is this correct?

Reply:
Since the other case calls compiled matlab, I'm guessing it is not correct.  I don't think `functionhighpass...` needs anything from the legacy fix settings, so the sourcing stuff shouldn't need to be kept.